ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/evaluator/AbstractEvaluatorTest.java

```diff
@@ -83,6 +83,14 @@ protected void testEvaluate(String fileName) throws BallerinaShellException {
 
                 ShellCompilation shellCompilation = evaluator.getCompilation(testCase.getCode());
                 Optional<PackageCompilation> compilation = shellCompilation.getPackageCompilation();
+
+                if (compilation.isEmpty() && testCase.getStderr().size() > 0) {
```

Comment:
Can be replace with,
```suggestion
                if (compilation.isEmpty() && !testCase.getStderr().isEmpty()) {
```